olm/olmconfig.yaml

```diff
@@ -22,13 +25,13 @@ description: |-
   This controller is a component of the [AWS Controller for Kubernetes](https://github.com/aws/aws-controllers-k8s)
   project. This project is currently in **developer preview**.
 samples:
-- kind: ExampleCustomKind
+- kind: EventBus
   spec: '{}'
```

Comment:
Question on syntax here: am I supposed to fill this one out/provide an example?

Reply:
I have never seen someone doing it before, but i guess it would be nice! After merging this the examples will make it to operator hub like s3 in https://operatorhub.io/operator/ack-s3-controller

Reply:
For ACK, these have been empty since the CR's change overtime before a controller is GA. The main reason is that for the embedded OperarorHub in  OpenShift, when installing the operator there is a `form view` which lists all the attributes of the CR and which ones are required and such. 

Maybe once a controller is GA, we can go an provide examples, but I don't feel that's necessary.